package.json

```diff
@@ -35,12 +36,18 @@
   },
   "devDependencies": {
     "@parcel/transformer-sass": "^2.0.1",
+    "@types/jest": "^27.4.0",
```

Comment:
27.4.7 is current. We probably needn't change this, as the most current patch version is picked anyway.

Reply:
Done